pep/translations/en-us.json

```diff
@@ -616,8 +616,7 @@
         },
         "justification": {
             "left": "Left",
-            "center": "Center",
-            "right": "Right"
+            "justify": "Left-Right Justified"
```

Comment:
i would label this as just "Justified" as thats how its referred to in virtually any other app, e.g. MS Word.

Reply:
I just put what its labeled as in the doc, but that's fine too